blockchain/baseProxy.go

```diff
@@ -227,6 +232,45 @@ func (proxy *baseProxy) GetRestAPIEntityType() core.RestAPIEntityType {
 	return proxy.endpointProvider.GetRestAPIEntityType()
 }
 
+// ProcessTransactionStatus will parse the provided transaction info and return its status accordingly
+// TODO move this in proxy, directly
+func (proxy *baseProxy) ProcessTransactionStatus(txInfo data.TransactionInfo) transaction.TxStatus {
+	if txInfo.Data.Transaction.Status != string(transaction.TxStatusSuccess) {
```

Comment:
Extract `txInfo.Data.Transaction.Status` in a variable

Reply:
will be erased, eventually. Will keep it like this for now :D